pkg/controller/replication/replication_controller.go

```diff
@@ -565,6 +565,12 @@ func (rm *ReplicationManager) syncReplicationController(key string) error {
 	}
 	rc := *obj.(*api.ReplicationController)
 
+	if rc.Spec.Paused {
```

Comment:
note that this is the rc from the store, so it might lag reality. I don't know if that matters.